metaseq/criterions/__init__.py

```diff
@@ -31,3 +31,10 @@ def build_criterion(cfg: DictConfig, task):
     if file.endswith(".py") and not file.startswith("_"):
         file_name = file[: file.find(".py")]
         importlib.import_module("metaseq.criterions." + file_name)
+
+criterions_dir = os.path.dirname(__file__).replace('metaseq/metaseq', 'metaseq-internal/metaseq_internal')
```

Comment:
What if we just did:

```
try:
    import metaseq_internal.criterions
except ImportError:
    pass
```

And we move all this dynamic importing into `metaseq_internal.criterions.__init__.py`?

Ditto for other approaches.

Reply:
Right, much clear :)

I added in the metaseq-inernal.{task/model}.__init__ references to metaseq's TASK_REGISTRY and MODEL_REGISTRY